### DISCUSSION TITLE
Unable to join the telegram group of Future Ready Talent program

### DISCUSSION THREAD

**Discussion by ananya555** [github/india#197] [Students]
I'm unable to join the telegram group, please send the link again

**Comment by ananya555** [comment]
When will this program get started?

**Reply by Akhil-Tony** [reply]
is there a dedicated telegram group for the same

**Reply by sachin23patel** [reply]
I also faces same problem

**Comment by TheSuranaverse** [comment]
@ananya555 @Akhil-Tony @sachin23patel 
**FUTURE READY SEPARATE DISCORD CHANNEL**

For smooth communication & discussion!
Link: https://discord.gg/ceMXzhfaka

Thank you 🙏
